### PR TITLE
SONAR-7493 Analysis can fail if message of issue has multibyte characters (Oracle DB)

### PR DESCRIPTION
**Pull request by simonbrandhof**
SONAR-7493 Analysis can fail if message of issue has multibyte characters (Oracle
DB)

### CODE REVIEW

**Comment by sns-seb** [comment]
commit message is too long

File: sonar-db/src/main/java/org/sonar/db/version/v56/FixLengthOfIssuesMessageOnOracle.java [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+  public FixLengthOfIssuesMessageOnOracle(Database db) {
+    super(db);
+  }
+
+  @Override
+  public void execute(Context context) throws SQLException {
+    if (getDatabase().getDialect().getId().equals(Oracle.ID)) {
+      // in order to not depend on value of variable NLS_LENGTH_SEMANTICS, unit of length
+      // is enforced to CHAR so that we're sure that type can't be 4000 BYTE.
+      context.execute("ALTER TABLE issues MODIFY (message VARCHAR (4000 CHAR))");
```
Comment (sns-seb) [inline sonar-db/src/main/java/org/sonar/db/version/v56/FixLengthOfIssuesMessageOnOracle.java]: I don't thihnk ISSUES#MESSAGE is our only column in DB with a varchar type, why fix only this one?

**Comment by sns-seb** [comment]
code LGTM, I let @eric-duquesnoy test the migration

File: sonar-db/src/main/java/org/sonar/db/version/v56/FixLengthOfIssuesMessageOnOracle.java (hunk 1 above)
Comment (simonbrandhof) [inline sonar-db/src/main/java/org/sonar/db/version/v56/FixLengthOfIssuesMessageOnOracle.java]: Done on purpose. Other columns will be fixed in 6.x : https://jira.sonarsource.com/browse/SONAR-7638

**Comment by eric-duquesnoy** [comment]
Migration has been successfully done.

**Pull request closed without merging**